### PR TITLE
mkfs : add boundary-align option

### PR DESCRIPTION
**Pull request by mat-c**
This allow to align fat and start of cluster on a boundary.
This may usefull for some media like sdcard.

Signed-off-by: Matthieu CASTET <castet.matthieu@free.fr>

### CODE REVIEW

File: mkfs/mkfs.c [hunk 1]
```diff
@@ -609,6 +617,17 @@ int main(int argc, char *argv[])
 			}
 			ui.cluster_size = ret;
 			break;
+		case 'b':
+			ret = parse_size(optarg);
+			if (ret < 0)
+				goto out;
+			else if (ret & (ret - 1)) {
```
Comment (namjaejeon) [inline mkfs/mkfs.c:631]: Right. maybe, It is needed for cluster size, if yes, can you send another patch for this ?

File: mkfs/mkfs.c [hunk 2]
```diff
@@ -416,6 +417,7 @@ static void usage(void)
 static struct option opts[] = {
 	{"volume-label",	required_argument,	NULL,	'L' },
 	{"cluster-size",	required_argument,	NULL,	'c' },
+	{"boundary-align",	required_argument,	NULL,	'c' },
```
Comment (namjaejeon) [inline mkfs/mkfs.c]: 'c' => 'b'

**Comment by namjaejeon** [comment]
Looks good to me:), Can you send a pull request to #exfat-next of https://github.com/exfatprogs/exfatprogs ?

**Comment by namjaejeon** [comment]
Need to update README and manpages also.

**Comment by mat-c** [comment]
Thanks for your comment. I also found a bug with some cluster/boundary.
I will post a new commit tomorrow.
